tests/inference/qnn_test.py

```diff
@@ -40,12 +40,14 @@ class QuantumInferenceTest(parameterized.TestCase, tf.test.TestCase):
   def setUp(self):
     """Initializes test objects."""
     super().setUp()
+    self.python_random_seed = 11
     self.tf_random_seed = 10
+    self.tf_random_seed_alt = 201
```

Comment:
does this mean we need two tf seeds for the tests to all pass?

Reply:
Added as a comment